client/src/components/SignUp/signup.style.js

```diff
@@ -32,7 +32,7 @@ export const StyledLabel = styled.label`
 `;
 
 
-export const StyledSignUp = styled.label`
+export const StyledLogin = styled.label`
```

Comment:
This is the signup styles? Look at comment above. You can extend like this:

`export const StyledSignUp = styled(StyledLogin)`